components/sensor/ade7953.rst

```diff
@@ -15,51 +15,226 @@ ADE7953 Power Sensor
 
 The ``ade7953`` sensor platform allows you to use ADE7953 single phase energy metering ICs
 (`datasheet <https://www.analog.com/media/en/technical-documentation/data-sheets/ADE7953.pdf>`__)
-with ESPHome. These are commonly found in **Shelly 2.5** devices.
+with ESPHome. These are commonly found in **Shelly** devices.
 
 This sensor can measure voltage and has two channels for reading current and active power (A & B).
 
 .. note::
 
     SAFETY HAZARD: Some devices such as Sonoff POWs/Shelly/etc, have the digital GND connected directly to mains voltage so **the GPIOs become LIVE during normal operation**. Our advice is to mark these boards to prevent any use of the dangerous digital pins.
 
-The :ref:`I²C Bus <i2c>` is
-required to be set up in your configuration for this sensor to work.
+The :ref:`I²C Bus <i2c>`
+or :ref:`SPI Bus <spi>`
+is required to be set up in your configuration for this sensor to work.
+
+Over I²C
+--------
+
+.. note::
+
+    Previously the platform name was _ade7953_. This name can still but _ade7953_i2c_ is recommended.
```

Comment:
I would prefer to remove all references to the old un-suffixed component as it may get removed in the future